package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service",
-  "version": "2.0.1",
+  "version": "2.1.0",
   "description": "Service side of clearlydefined.io.",
```

Comment:
probably want to rebase before putting up the PR.  There shouldn't be any changes in package.json or package-lock.json